package.json

```diff
@@ -17589,7 +17589,12 @@
 			{
 				"view": "gitlens.views.worktrees",
 				"contents": "DevEx Days 24 Sale: Save up to 80% on GitLens Pro - lowest price of the year!",
-				"when": "gitlens:plus:required && gitlens:plus:state == 4 && gitlens:promo == devexdays24"
+				"when": "gitlens:plus:required && (gitlens:plus:state == 4 || gitlens:plus:state == 7) && gitlens:promo == devexdays24"
```

Comment:
I think we would want to apply `gitlens:plus:state == 7` case to the "pro 50% off" promo just above this as well.

Reply:
I thought it was only for free accounts that converted to pro accounts for the first time. Since you come from an expired pro account, I thought it wouldn't make sense to make a promo on your first seat, as you're not new here. But I may be missing some information so you may be right :+1:

Reply:
Ah yes, good point! I hadn't thought it through. Then let's leave it as you had it, without the pro50 promo included. 👍🏼 Sorry for the unnecessary change.